.github/workflows/parameters_check.yml

```diff
@@ -24,7 +24,6 @@ jobs:
         with:
           repository: malb/lattice-estimator
           path: lattice_estimator
-          ref: '53508253629d3b5d31a2ad110e85dc69391ccb95'
```

Comment:
could we pin the more recent reference ? it is to avoid random breakage in the CI and means we have to dedicate some time from time to time to update the commit

Reply:
Ok, I'll add the current commit back.

Reply:
Added back on L27